### DISCUSSION TITLE
E-reader support and Joplin bridge

### DISCUSSION THREAD

**Discussion by Szybet** [tsujan/FeatherNotes#305] [General]
Hello
Great project, good code, thanks for your work.
I would want to share a few things that I did related to this app and maybe it could be added to the readme?

So, [this project](https://github.com/Szybet/feather-joplin-bridge), a bridge between joplin and this app, for now it can only convert Joplin notes to feather format, but i plan to expand it. And I ported [FeatherNotes to InkBox](https://github.com/Szybet/FeatherNotes-inkbox), an e-reader OS and it works great.

**Comment by tsujan** [comment]
Thanks for the info! I'm converting it to a Discussion...
